src/sonic-bgpcfgd/bgpmon/bgpmon.py

```diff
@@ -106,11 +104,10 @@ def flush_pipe(self, data):
         for key, value in data.items():
             if value is None:
                 # delete case
-                self.pipe.delete(key)
+                self.db.delete(self.db.STATE_DB, key)
             else:
                 # Add or Modify case
-                self.pipe.hmset(key, value)
-        self.pipe.execute()
+                self.db.hmset(self.db.STATE_DB, key, value)
```

Comment:
Will improve this code with pipe

Reply:
Fixed, now this PR depends on https://github.com/Azure/sonic-swss-common/pull/647
We need that PR finish and update submodule first.

Reply:
sonic-swss-common PR merged, create another PR to merge submodule: https://github.com/Azure/sonic-buildimage/pull/11470

After that, this PR can pass UT.